internal/services/firewall/firewall_resource_test.go

```diff
@@ -25,6 +25,61 @@ const (
 	standard = "Standard"
 )
 
+func TestAccFirewall_autoscaleConfigMinMax(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_firewall", "test")
+	r := FirewallResource{}
+	minScale := 3
+	maxScale := 3
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.withAutoscaleConfiguration(data, &minScale, &maxScale),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("autoscale_configuration.0.min_capacity").HasValue("3"),
+				check.That(data.ResourceName).Key("autoscale_configuration.0.max_capacity").HasValue("3"),
```

Comment:
These checks can be removed, as they are covered by the test framework internally.